employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/rotation/view/ShiftTemplateView.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.domain.rotation.view;
+
+import java.time.Duration;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.optaweb.employeerostering.domain.common.AbstractPersistable;
+import org.optaweb.employeerostering.domain.common.HasTimeslot;
+import org.optaweb.employeerostering.domain.rotation.ShiftTemplate;
+
+public class ShiftTemplateView extends AbstractPersistable implements HasTimeslot {
```

Comment:
Do I understand it correctly that *View classes are equivalent to the *DTO classes in terms of their purpose? If so, it would be good to unify the naming and place them to a similar package.

Reply:
The *View classes and *DTO classes don't have the same purpose. The DTO classes were newly added to avoid the security flaw mentioned above and conform with Spring convention. From what I understand, the View objects were used to store data useful for the _old_ frontend to display the information.

@Christopher-Chianelli feel free to correct me if my comments above are wrong. Does the new frontend require the *View objects?

Reply:
The `*DTO` and `*View` classes share the same purpose (which is to remove backend specific information such as time zone that is not needed in the frontend). The new frontend require the *View objects. (Although the `*DTO` were introduced to circumvent a security flaw, that security flaw is related to the purpose of the `*View`)

Reply:
I see. Would it make sense to rename the `*DTO` classes to `*View` for consistency?

Reply:
+1 to renaming `*DTO` classes to `*View`

Reply:
If the purpose is to get the data to frontend, `*View` sounds good.